### DISCUSSION TITLE
Stuck on Install.ps1

### DISCUSSION THREAD

**Discussion by DoisPontinhos** [LSPosed/MagiskOnWSALocal#278] [Q&A] [answered]
Hey guys, I'm stuck on Install.ps1 and getting this message:

![powershell_z557hXnwEJ](https://user-images.githubusercontent.com/82269167/206326925-ddf12e50-cd2c-4144-978a-d798425e89d6.png)

**Comment by BiggDaddyy** [comment]
omg SAME 0,o

**Comment by Howard20181** [comment, marked as answer]
Not a bug.
WSA cannot be installed from a network location, you have to copy it to the Windows directory instead of installing it directly from the WSL directory.

**Comment by bentahsin** [comment]
![image](https://user-images.githubusercontent.com/75451952/230772814-cb588cd8-a23f-476f-a2e4-16aada959ba3.png)
same
